README.md

```diff
@@ -44,6 +62,14 @@ npm run build
 
 This command generates static content into the `build` directory and can be served using any static contents hosting service.
 
+### Import documentation
+
+```bash
+npm run import-docs
+```
+
+This command downloads the latest markdown files from the [development-standards](https://github.com/Amsterdam/development-standards) repository. By default this will save documentation in the `docs` directory; to customize this you can pass as argument of the desired path after the `import-docs` command.
```

Comment:
as or an? argument of the desired ...

Reply:
Good spot, cleared up this sentence a bit